provider/src/endpoints/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Interaction, InteractionObject, Matchers, RequestOptions, ResponseOptions } from '@pact-foundation/pact';
```

Comment:
feedback:

As I wrote above this only works within the same tech stack and a monorepo, not if the provider is written in python or java and in an own repository.